lib/entities/content-type.ts

```diff
@@ -195,7 +196,7 @@ export interface ContentType
 
 function createContentTypeApi(http: AxiosInstance): ContentTypeApi {
   return {
-    update: createUpdateEntity({
+    update: createUpdateEntity<ContentType>({
```

Comment:
Can `ContentType` be inferred from the wrapperMethod?

Reply:
I wouldn't get it to work easily but I'll give it another try.

Reply:
Something along these lines:

```
type WrapperMethod<E, A extends AxiosInstance = AxiosInstance> = (http: A, data: any) => E;

export function createUpdateEntity<T>({}: {
  wrapperMethod: WrapperMethod<T>
})
```

Reply:
done! a good suggestion 👍